src/test/autodjprocessor_test.cpp

```diff
@@ -124,10 +127,10 @@ class FakeDeck : public BaseTrackPlayer {
 class MockPlayerManager : public PlayerManagerInterface {
   public:
     MockPlayerManager()
-            : numDecks(ConfigKey("[Master]", "num_decks"), true),
-              numSamplers(ConfigKey("[Master]", "num_samplers"), true),
-              numPreviewDecks(ConfigKey("[Master]", "num_preview_decks"),
-                              true) {
+            : numDecks(ConfigKey(kAppGroup, QStringLiteral("num_decks")), true),
+              numSamplers(ConfigKey(kAppGroup, QStringLiteral("num_samplers")), true),
+              numPreviewDecks(ConfigKey(kAppGroup, QStringLiteral("num_preview_decks")),
+                      true) {
```

Comment:
Please only add a test for the new CO name and leave the existing one as it is

Reply:
The plan was to remove the word from the codebase as much as possible which includes tests. Also, these tests are supposed to check if the AutoDJ processor works correctly, not if old CO names still work. I agree that it makes sense to test this, too, but that should be a different test IMHO. 

I was planning to add a new test that checks if the aliases exist. This would ideally go into `coreservicestest.cpp` since most other tests do not create all COs. Unfortunately this test is broken at the moment. I'm currently looking into it.

Reply:
this would / could result in warnings raised by tests though if we decided to implement the warnings on deprecated CO access.

Reply:
We need to ensure the backward compatibility of our legacy API. I'm open for any test implementation, that ensures this.